test/useWith.js

```diff
@@ -8,12 +8,12 @@ describe('useWith', function() {
   function add1(x) { return x + 1; }
   function mult2(x) { return x * 2; }
   function div3(x) { return x / 3; }
-  var f = R.useWith(max, add1, mult2, div3);
+  var f = R.useWith(max, [add1, mult2, div3]);
 
   it('takes a arbitrary number of function arguments and returns a function', function() {
```

Comment:
@davidchambers is this description as brief as it could be?
--> it('nonetheless has the correct arity', function() {
    assert.strictEqual(f.length, 3);
  });
--> perhaps nonetheless is superfluous ;)